### ISSUE TITLE
More Tracks?

### ISSUE THREAD

**Issue by Erani0** [sati5M/GoKarts#1]
Any Options for add Multitasking with multi races same time?

**Comment by sati5M** [comment]
Unfortunately, this doesn't exist at this time and I don't think I'll be adding it any time soon.

**Comment by anes66** [comment]
> 

yeah bro it would be nice out there there are more maps for karting
